packages/web-components/src/components/va-omb-info/va-omb-info.tsx

```diff
@@ -50,6 +50,12 @@ export class VaOmbInfo {
     this.modalContents = null;
   };
 
+  // Redirects focus back to the modal, if the modal is open/visible
+  private trapFocus = () => {
+    const modalCloseButton = this.el.shadowRoot.querySelector('va-modal').shadowRoot.querySelector('button.va-modal-close') as HTMLElement;
```

Comment:
This is kooky nit feedback but could this be formatted the same as what you've done for the `va-crisis-line-modal` version of the same method?